tests/Doctrine/Tests/DBAL/Platforms/MariaDb102PlatformTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Platforms;
+
+use Doctrine\DBAL\Platforms\MariaDb102Platform;
+use Doctrine\DBAL\Schema\Comparator;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Types\Type;
+
+class MariaDb102PlatformTest extends AbstractMySQLPlatformTestCase
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function createPlatform()
```

Comment:
Missing return types for all methods in this class.

Reply:
Thanks. Fixed for all methods